src/ui/layout/navbar.svelte

```diff
@@ -1,31 +1,38 @@
 <script lang="ts">
   import { Theme } from "../theme/theme";
-  import NavBarHeader from "./navbar-header.svelte";
-  import SectionLayout from "./section-layout.svelte";
   import type { BrandingAppearance } from "../../entities/branding";
+  import type { Snippet } from "svelte";
 
-  export let brandingAppearance: BrandingAppearance | null | undefined =
-    undefined;
-  $: style = new Theme(brandingAppearance).productInfoStyleVars;
+  type Props = {
+    headerContent?: Snippet<[]>;
+    bodyContent?: Snippet<[]>;
+    brandingAppearance?: BrandingAppearance | null;
+  };
 
-  export let headerContent;
-  export let bodyContent: () => any;
+  const {
+    headerContent,
+    bodyContent,
+    brandingAppearance = undefined,
+  }: Props = $props();
 
-  export let showCloseButton: boolean;
-  export let onClose: (() => void) | undefined = undefined;
+  const style = $derived(new Theme(brandingAppearance).productInfoStyleVars);
 </script>
 
 <div class="rcb-ui-navbar" {style}>
-  <SectionLayout layoutStyle="justify-content: flex-end;">
-    {#snippet header()}
-      <NavBarHeader {showCloseButton} {onClose}>
```

Comment:
no more `onClose`?

Reply:
oh, I see is under Branding Info

Reply:
Yes but I moved it to a different place. Although I think you're right and here seems a better fit. Let's see if I can fix it.

Reply:
I have renamed `BrandingInfo` to `BrandingHeader` which is more accurate.